src/main/resources/static/js/MainWindow.js

```diff
@@ -360,7 +360,7 @@ function saveConfig() {
 							}
 						});
 		return false;
-	} else if (streams.length === 0) {
+	} else if (streams.length == 0) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use ===/!== to compare with true/false or Numbers](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=4872216)